### ISSUE TITLE
Loss function for YOLO NAS

### ISSUE THREAD

**Issue by rohankhaire-work** [Deci-AI/super-gradients#2075]
### 💡 Your Question

Hello,

I am trying to implement loss function for YOLO NAS model in PyTorch to learn.
Can anyone please tell me about the loss functions used in YOLO NAS model.

The data that is available to me is -
- ground truth bounding boxes x, y, w, h
- ground truth labels for classes

Based on what I know -
- The forward pass of the model gives the **bounding boxes** and **confidence scores** in the first tuple and there are **logits** in the second tuple along with some more data that I do not understand.
- So, we need to calculate regression loss for the bounding boxes and the classification loss for the labels.
- I am not sure if there is any other loss that is needed to be calculated given my data.

Could you please let me know if I am missing anything?
